utils/asyncQueue.ts

```diff
@@ -12,13 +13,17 @@ import { nextTick } from "./nextTick.ts";
  * }
  * ```
  */
+
+
+
 export class AsyncQueue<T> {
   private queue: T[] = [];
```

Comment:
Last time I checked, which admittedly is quite a long time ago, `Array.prototype.shift()` is very, very slow. It would be better to use a linked list here, just like we did in https://github.com/RubenVerborgh/AsyncIterator/pull/46 (very similar scenario).

Reply:
Just merged it: https://github.com/seriousme/opifex/pull/52
Thanks,
Hans